x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -163,6 +237,31 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
       },
       sortable: true,
     },
+    {
+      'data-test-subj': 'aiOpsSpikeAnalysisTableColumnAction',
+      name: i18n.translate('xpack.aiops.spikeAnalysisTable.actionsColumnName', {
+        defaultMessage: 'Actions',
+      }),
+      actions: [
+        {
+          name: () => (
+            <EuiToolTip content={discoverUrlError ? discoverUrlError : viewInDiscoverMessage}>
+              <EuiIcon type="link" />
+            </EuiToolTip>
+          ),
+          description: viewInDiscoverMessage,
+          type: 'button',
+          onClick: async (changePoint) => {
+            const openInDiscoverUrl = await generateDiscoverUrl(changePoint);
+            if (typeof openInDiscoverUrl === 'string') {
+              await application.navigateToUrl(openInDiscoverUrl);
+            }
+          },
+          enabled: () => discoverUrlError === undefined,
+        },
+      ],
+      width: '60px',
```

Comment:
Nit: For the other columns we already have a constant with the custom width (`NARROW_COLUMN_WIDTH`). Even if this is just one other column, suggest to create another constant like `ACTION_COLUMN_WIDTH` for this.

Reply:
Updated in 344f7b0db372c97d3351771339e4fcccc9c789ff